src/es/components/atoms/breadcrumb/Breadcrumb.js

```diff
@@ -33,10 +33,15 @@ export default class Breadcrumb extends Shadow() {
    * @return {void}
    */
   renderCSS () {
+    // extract frequently used value to variables
+    const icon = this.getAttribute('icon')
+    const color = 'var(--color, pink)'
+    const textDecoration = 'var(--text-decoration, none)'
+
     this.css = /* css */`
       :host {
-        margin: var(--margin, 0) !important;
-        width: var(--width, 0) !important;
+        margin: var(--margin, 0); /* removed "!important" and added a default value */
+        width: var(--width, 0); /* removed "!important" and added a default value */
```

Comment:
this margin and width with fallback 0 is questionable and possible not needed. needs to be testet... if needed for some reason leave !important to overseed the width and margin being set by o-body to all children within main. Also !important is not as ugly within a webcomponent as it used to be with a global only stylesheet.

Reply:
🤔 You mean in the case o-body is set for breadcrumb AND other components together? Then I will roll back it again.